crates/sncast/tests/helpers/fixtures.rs

```diff
@@ -359,6 +355,41 @@ pub fn duplicate_script_directory(
     let mut file = File::create(manifest_path).expect("Failed to create file");
     file.write_all(modified_toml.as_bytes())
         .expect("Failed to write to file");
+}
+
+pub fn duplicate_script_directory(
+    src_dir: impl AsRef<Utf8Path>,
+    deps: Vec<impl AsRef<std::path::Path>>,
+) -> TempDir {
+    let temp_dir = copy_directory_to_tempdir(&src_dir);
+
+    let dest_dir = Utf8PathBuf::from_path_buf(temp_dir.path().to_path_buf())
+        .expect("Failed to create Utf8PathBuf from PathBuf");
+
+    copy_script_directory(&src_dir, dest_dir, deps);
+
+    temp_dir
+}
+
+pub fn duplicate_workspace_directory(
```

Comment:
why do we need this? by the name I assume you just want to copy a directory, so why not use `fs` or some other crate?

Reply:
We want to copy the specified members and their deps, including `sncast_std` (use `duplicate_script_directory` for every member)

Reply:
I'm not sure why we need a distinction between `script` and `workspacez directory, it seems to me this could be a one function tbh

Reply:
It doesn't seem to me we can merge it into one function, they have different structures: for workspace we need to specify `members`, where each member is a script, which is something we don't do for scripts.

Reply:
hmm, ok I'm convinced. :+1: